tools/sklearn/pipeline.xml

```diff
@@ -0,0 +1,130 @@
+<tool id="sklearn_pipeline" name="Build a Pipeline" version="@VERSION@">
```

Comment:
should we use a more specific tool_id? Maybe something like `sklearn_model_selection` and the name could also be more meaingfiul imho.

Reply:
or sklearn_search_model_selection?